src/demo/DemoTestLoanService.java

```diff
@@ -2267,4 +2269,43 @@ private static void testBulkReverseLoanTransactions() throws MambuApiException {
 		return loanTransactions;
 	}
 
+	/**
+	 * Tests PATCHing the setExpectedDisbursementDate and setFirstRepaymentDate on a loan account.
+	 * 
+	 * @throws MambuApiException
+	 */
+	private static void testPatchLoanAccountDisbursementAndFirstRepaymentDate() throws MambuApiException {
+
+		methodName = new Object() {}.getClass().getEnclosingMethod().getName();
+		System.out.println("\nIn : " + methodName);
+
+		LoansService loanService = MambuAPIFactory.getLoanService();
+
+		LoanAccount theAccount = newAccount;
+		LoanAccount account = new LoanAccount();
+
+		account.setId(theAccount.getId());
+
+		Calendar currentCalendar = Calendar.getInstance();
+		currentCalendar
+				.setTime(theAccount.getDisbursementDate() != null ? theAccount.getDisbursementDate() : new Date());
+		currentCalendar.add(Calendar.DAY_OF_MONTH, 1);
+
+		account.setExpectedDisbursementDate(currentCalendar.getTime());
+
+		currentCalendar.add(Calendar.DAY_OF_MONTH, 2);
+		account.setFirstRepaymentDate(currentCalendar.getTime());
+
+		BigDecimal nullBigDecimal = null;
+		// null LoanAmount and RepaymentInstallments fields, they don't need to be present in the PATCH JSON
+		account.setLoanAmount(nullBigDecimal);
+		account.setRepaymentInstallments(null);
+
+		// patch the loan account
+		boolean patchResult = loanService.patchLoanAccount(account);
+
+		System.out.println("The expectedDisbursementDate and firstRepaymentDate were edited for the loan account "
+				+ NEW_LOAN_ACCOUNT_ID + ". Status: " + patchResult);
+	}
+
```

Comment:
In general, for testing bugs issues if we can try adding test scenarios to the existent "functional API" methods whenever possible that would be best (may be test setting these dates using an existent testPatchLoanAccountTerms()?) , without creating dedicated bugs test methods